src/features/projectsV2/ProjectsMap/ProjectSiteDropDown/SiteDropdown.module.scss

```diff
@@ -67,9 +62,8 @@ $layoutPaddingSide: 20px;
   width: 350px;
   font-size: $fontXSmall;
   position: absolute;
-  z-index: 50;
-  top: 84px;
-  right: 30px;
+  z-index: 99;
+  top: 44px;
```

Comment:
Perhaps this should be a calculated value so that it is clear why we're getting a figure of 44px